tests/apps/bookinfo/output/productpage-normal-user-v2.html

```diff
@@ -0,0 +1,178 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Simple Bookstore App</title>
+<meta charset="utf-8">
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+<!-- Latest compiled and minified CSS -->
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+
+<!-- Optional theme -->
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap-theme.min.css">
+
+  </head>
+  <body>
+    
+    
+
+<nav class="navbar navbar-inverse navbar-static-top">
+    <div class="container">
+        <div class="navbar-header">
+            <a class="navbar-brand" href="#">BookInfo Sample</a>
+        </div>
+        
+        <p class="navbar-text navbar-right">
+            <i class="glyphicon glyphicon-user" aria-hidden="true"></i>
+            <span style="padding-left: 5px;">normal-user  ( <a href="logout">sign out</a> )</span>
+        </p>
+        
+    </div>
+</nav>
+
+<!---
```

Comment:
at least remove that block ?

Reply:
This is part of bookinfo implementation, the output html reflects the actual output of the application.